addons/website/views/snippets/s_comparisons_horizontal.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_comparisons_horizontal" name="Comparisons Horizontal">
+    <section class="s_comparisons_horizontal o_colored_level pt64 pb48">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12">
+                    <h3>Our Pricing</h3>
+                    <p class="lead">Discover what is the best pricing for you.</p>
+                </div>
+                <div class="col-12 col-lg-6 pb16 pt16" data-name="Card">
+                    <div class="s_card card o_cc o_cc1 h-100 my-0" data-vxml="001" data-snippet="s_card" data-name="Card">
+                        <div class="card-body">
+                            <div class="row">
+                                <div class="col-6 shadow" style="box-shadow: rgba(0, 0, 0, 0.15) -1px 0px 0px 0px inset !important;">
+                                    <h5 class="card-title">Professional</h5>
```

Comment:
Going from `h3` (Our Pricing) to `h5` (card-title) does not seem optimal as you're skipping headings levels. I'd suggest this instead : 

`<h2 class="h3-fs">Our Pricing</h2>`

and 

`<h3 class="card-title h5-fs">Professional</h3>`

Could you adapt the other occurrences as well?